### PR TITLE
add works section

### PR DESCRIPTION
**Pull request by SekoViper**
add works section template.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -120,4 +120,136 @@ p {
...
+
+  .btn {
+    padding: 12px;
+    color: var(--primary-color);
+    font-size: 12px;
+    line-height: 16px;
+    border-radius: 8px;
+    border: none;
+    background-color: var(--simple-color);
+  }
```
Comment (ichala) [inline css/style.css:189]: - [ ] the button should follow the button styling: Active, Hover and Pressed kindly fix that like mentioned in Figma design 
image

File: index.html [hunk 2]
```diff
@@ -53,6 +53,120 @@ <h1>I’m Seko<br> Glad to <br>see you!</h1>
...
+                </div>
+            </div>
+
+
+        <section class="about-section">
+            <div class="about-container">
+                <div class="desc">
+                    <h2 class="about-heading">About<br> Myself</h2>
+                    <p class="about-desc">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    <a href="#" class="connect">Let's Connect</a>
```
Comment (ichala) [inline index.html:149]: - [ ] Please note that the `Let's Connect` title should be a link, instead it should be a `hx` title for the icons below it , kindly fix it

File: index.html [hunk 3]
```diff
@@ -53,6 +53,120 @@ <h1>I’m Seko<br> Glad to <br>see you!</h1>
...
+                    <div class="social-icon">
+                        <ul>
+                            <li><a href="#"><img src="/img/twiter icon.svg" alt="Twitter icon"></a></li>
+                            <li><a href="#"><img src="/img/Linkedin icon.svg" alt="LinkedIn Icon"></a></li>
+                            <li><a href="#"><img src="/img/medium icon.svg" alt="medium icon"></a></li>
+                            <li><a href="#"><img src="/img/github icon.svg" alt="github icon"></a></li>
+                            <li><a href="#"><img src="/img/hi icon.svg" alt="Hi icon"></a></li>
+                        </ul>
+                    </div>
+                    <button class="btn btn-primary">Get My Resume</button>
```
Comment (ichala) [inline index.html:159]: - [ ] the button should follow the button styling: Active, Hover and Pressed kindly fix that like mentioned in Figma design 
image

File: index.html [hunk 4]
```diff
@@ -53,6 +53,120 @@ <h1>I’m Seko<br> Glad to <br>see you!</h1>
...
+                </div>
+                <div class="skils">
+                    <ul class="skills-items">
+                        <li class="skills-item"><h3>Languages</h3> <img src="/img/Union.png" alt=""></li>
```
Comment (ichala) [inline index.html]: - [ ] kindly add The technology list ("JavasScript", "HTML" ..) in languages section like shown in figma

image

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @SekoViper , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Clean Code✅ 
- Greate Design✅ 
- Linters are passing ✅


 ## Required Changes ♻️



- [ ] _**check the comments below**_


## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (SekoViper) [reply index.html:149]: please I do not understand this pretty well. ==>"Please note that the Let's Connect title should be a link, instead it should be a hx title for the icons below it, kindly fix it"

File: index.html (hunk 4 above)
Comment (SekoViper) [reply index.html]: To get that done, I will need to use javascript that we are not supposed to use. form the project requirement page"You don't need to implement the detail modal window that requires JavaScript."

Screenshot 2022-09-22 135611

Comment (mckent05) [reply index.html]: Please note that Javascript is not needed to implement this 😄 . You do not necessarily need to collapse the Languages frame, for now it is okay to implement them just as they are in the Figma design 👍 👍

Comment (mckent05) [reply index.html]: Just like the previous reviewer suggested and as I replied, please ensure that the languages section is implemented in your design, for now you do not need to use Javascript to implement this section 😄 😄

File: index.html (hunk 3 above)
Comment (mckent05) [reply index.html:159]: - Good job implementing this change as suggested by the reviewer 👍 👍

File: css/style.css [hunk 5]
```diff
@@ -120,4 +120,152 @@ p {
...
+    display: grid;
+  }
+
+  .work-container {
+    background-color: #fff;
+    padding: 16px;
+    margin: 114px 24px 0 24px;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+  }
```
Comment (mckent05) [inline css/style.css]: **[OPTIONAL]** - Please note that for each of your works cards, the content of the project should be aligned with the `flexbox` property. This is a project requirement. 😄 😄

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @SekoViper, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your code.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by levy002 — APPROVED** [review]
### Hello @SekoViper 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**